assets/azurerm/icons.json

```diff
@@ -0,0 +1,462 @@
+{
+ "azurerm_advanced_threat_protection": "",
+ "azurerm_advisor_recommendations": "",
+ "azurerm_analysis_services_server": "",
+ "azurerm_api_management": "",
+ "azurerm_api_management_api": "",
+ "azurerm_api_management_api_operation": "",
+ "azurerm_api_management_api_operation_policy": "",
+ "azurerm_api_management_api_policy": "",
+ "azurerm_api_management_api_schema": "",
+ "azurerm_api_management_api_version_set": "",
+ "azurerm_api_management_authorization_server": "",
+ "azurerm_api_management_backend": "",
+ "azurerm_api_management_certificate": "",
+ "azurerm_api_management_diagnostic": "",
+ "azurerm_api_management_group": "",
+ "azurerm_api_management_group_user": "",
+ "azurerm_api_management_identity_provider_aad": "",
+ "azurerm_api_management_identity_provider_facebook": "",
+ "azurerm_api_management_identity_provider_google": "",
+ "azurerm_api_management_identity_provider_microsoft": "",
+ "azurerm_api_management_identity_provider_twitter": "",
+ "azurerm_api_management_logger": "",
+ "azurerm_api_management_named_value": "",
+ "azurerm_api_management_openid_connect_provider": "",
+ "azurerm_api_management_product": "",
+ "azurerm_api_management_product_api": "",
+ "azurerm_api_management_product_group": "",
+ "azurerm_api_management_product_policy": "",
+ "azurerm_api_management_property": "",
+ "azurerm_api_management_subscription": "",
+ "azurerm_api_management_user": "",
+ "azurerm_app_configuration": "",
+ "azurerm_app_service": "Containers/10035-icon-service-App-Services.svg",
+ "azurerm_app_service_active_slot": "",
+ "azurerm_app_service_certificate": "",
+ "azurerm_app_service_certificate_order": "",
+ "azurerm_app_service_custom_hostname_binding": "",
+ "azurerm_app_service_environment": "",
+ "azurerm_app_service_hybrid_connection": "",
+ "azurerm_app_service_plan": "",
+ "azurerm_app_service_slot": "",
+ "azurerm_app_service_slot_virtual_network_swift_connection": "",
+ "azurerm_app_service_source_control_token": "",
+ "azurerm_app_service_virtual_network_swift_connection": "",
+ "azurerm_application_gateway": "Other/10076-icon-service-Application-Gateways.svg",
+ "azurerm_application_insights": "Management + Governance/00012-icon-service-Application-Insights.svg",
```

Comment:
Format the names to not have spaces or characters. so basically `Management_and_Governance`